### PR TITLE
Version 1.9.3

### PR DESCRIPTION
**Pull request by Schmiddiii**

### CODE REVIEW

File: de.schmidhuberj.tubefeeder.json [hunk 1]
```diff
@@ -8,14 +8,13 @@
     ],
     "command": "tubefeeder",
     "finish-args": [
+        "--share=ipc",
         "--socket=fallback-x11",
         "--socket=wayland",
         "--socket=pulseaudio",
         "--device=dri",
         "--share=network",
         "--filesystem=xdg-data/tubefeeder:create",
```
Comment (hfiguiere) [inline de.schmidhuberj.tubefeeder.json:17]: why do you need this?

File: de.schmidhuberj.tubefeeder.json [hunk 2]
```diff
@@ -8,14 +8,13 @@
...
-        "--filesystem=xdg-cache/tubefeeder:create",
-        "--filesystem=xdg-config/mpv:ro",
         "--filesystem=xdg-download",
         "--talk-name=org.freedesktop.portal.FileChooser",
```
Comment (hfiguiere) [inline de.schmidhuberj.tubefeeder.json:19]: I don't think you need this one either. portal access is granted by default.

File: de.schmidhuberj.tubefeeder.json (hunk 1 above)
Comment (Schmiddiii) [reply de.schmidhuberj.tubefeeder.json:17]: This fixes the warning `finish-args-x11-without-ipc`. Don't really know what it is for though.

File: de.schmidhuberj.tubefeeder.json (hunk 2 above)
Comment (Schmiddiii) [reply de.schmidhuberj.tubefeeder.json:19]: Didn't know. I will try without.

**Comment by Schmiddiii** [comment]
Note that this is still waiting for https://github.com/flathub/flatpak-builder-lint/pull/89 as there is a new error when having `org.freedesktop.Flatpak` permission that I require.

File: de.schmidhuberj.tubefeeder.json (hunk 1 above)
Comment (hfiguiere) [reply de.schmidhuberj.tubefeeder.json:17]: it's necessary for X11 (or fallback). It should have been noted back then at review time, must have fallen through the cracks, the linter make this a hard error.

**Comment by Schmiddiii** [comment]
bot, build

**Pull request merged**